README.md

```diff
@@ -28,3 +28,7 @@ Creating a pre-release for this repo will automatically update the `testing` tag
 ### Updating `stable` tag
 
 Creating a release for this repo will automatically update the `stable` tag used by production apps.
+
+### Creating terraforn-docs
+
+To simply add [terraform-docs](https://terraform-docs.io/), run 'docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.16.0 markdown /terraform-docs > tfdocs.md' in the module directory to create a terraform-docs file, tfdocs.md.  (leave out  '-u $(id -u)'  when running on windows)
```

Comment:
I reckon something like this will make it clearer:

```suggestion
## Updating [Terraform Docs](https://terraform-docs.io/)

Run the following command in each module directory where the docs need to be updated:

```sh
docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.16.0 markdown /terraform-docs > tfdocs.md
```

The `-u $(id -u)` can be left out when running on Windows.
```